appmonitoring/ts/src/CertificateManager.ts

```diff
@@ -225,53 +258,65 @@ export class CertificateManager {
     }
 
     private async IsValidCertificate(operationId: string, mwhcCaBundle: string, webhookCertData: WebhookCertData, clusterArmId: string, clusterArmRegion: string): Promise<boolean> {
+        if (webhookCertData == null)
+        {
+            logger.info('WebhookCertData is null', operationId, this.requestMetadata);
+            await logger.SendEvent("CertificateFetchValueFailure", operationId, null, clusterArmId, clusterArmRegion, true);
+            return false;
+        }
+
         try {
             forge.pki.certificateFromPem(mwhcCaBundle);
             forge.pki.certificateFromPem(webhookCertData.caCert);
+            forge.pki.privateKeyFromPem(webhookCertData.caKey);
```

Comment:
Let's cover the scenario where the certs are NOT regenerated (all good, nothing to do) with a unit test so that this can't be missed